docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,294 @@
+= Service Maturity
+
+Implementing and maintaining a service is iterative work.
+The idea is to release early and release often, without having the urge to have all features available right from the beginning.
+
+This page gives an idea on the iterative steps and a way to assess the service maturity.
+Except the "Provisioning" iteration and the stable release, all steps in-between can be in any order, as it fits best.
+With "feature complete" it's not meant that the service doesn't receive new functionality or doesn't get maintained anymore, we're talking about the default scope outlined on this page.
+
+Important is that every step is independent of each other. A service that implements `Backup and Restore` doesn't necessarily already implement `Logs`.
+That in turn also means we must allow some leeway for each service in respect to the requirements, should a service not allow for a feature or makes it overly complex.
+A service is allowed to deviate from a requirement when it makes sense and can be documented and reasoned about accordingly.
+
+TODO! To understand the personas referenced in this document, see xref:reference/glossary.adoc#_personas[Glossary - Personas].
+
+TODO! We currently maintain the service maturity assessment for each service on our product documentation site under https://products.docs.vshn.ch/products/appuio/managed/services_index.html[VSHN Application Catalog -> Services^].
+
+== Initial Release: Cluster Provisioning
+
+As a user of the service I can perform the following actions:
+
+Provisioning::
+I can provision a production-ready cloud Kubernetes environment on a major cloud service provider, such as Amazon EKS, Azure AKS, Google GKE, Exsoscale SKS (on request) and Swisscom ESC (on request).
```

Comment:
```suggestion
I can provision a Cloud Kubernetes environment on our supported Cloud Service Provider.
```

I'd leave "production-ready" out here, it's unclear what this actually means. Maintaining the list of support Cloud Service Providers shouldn't be done here, instead it's part of products.docs.

Reply:
Oh, and we have to bring in the "self-service" aspect here somehow.

Reply:
For us (colin & Hadar) it is clear it is a self-service. because it is referring to the End User capabilities.

Reply:
Always be explicit in such documents, don't make assumption. For me, it wasn't clear, and therefore I strongly suggest being precise. That's why I propose to bring the self-service aspect into it, as this is one of the most important pillars of the product.